TerminalDocs/tutorials/custom-prompt-setup.md

```diff
@@ -3,148 +3,138 @@ title: Windows Terminal Custom Prompt Setup
+To customize your PowerShell prompt, you can install Oh My Posh using the [PowerShell install-module](/powershell/module/powershellget/install-module). Enter the command:
 
-    ```powershell
-    Get-PoshThemes
-    ```
+```powershell
+Install-Module oh-my-posh -Scope CurrentUser
+```
 
-3. Choose a theme and update your PowerShell profile with this command. (You can replace `notepad` with the text editor of your choice.)
+This will install:
 
-    ```powershell
-    notepad $PROFILE
-    ```
+- `oh-my-posh.exe`: The Windows executable
+- `themes`: The latest [Oh My Posh themes](https://ohmyposh.dev/docs/themes)
 
-4. Add the following to the end of your PowerShell profile file to set the `paradox` theme. (Replace `paradox` with the theme of your choice.)
+You may also want to enter this command to ensure you have the latest updates: `Update-Module oh-my-posh`.
 
-    ```powershell
-    Import-Module oh-my-posh
-    Set-PoshPrompt -Theme paradox
-    ```
+> [!NOTE]
+> If you are more familiar with the [Scoop](https://scoop.sh/) installer or [Chocolatey](https://chocolatey.org/) package manager, these can also be used for installing on Windows, just follow the instructions in the [Oh My Posh docs](https://ohmyposh.dev/docs/windows).
 
-Now, each new PowerShell instance will start by importing Oh My Posh and setting your command line theme.
+### Choose and apply a PowerShell prompt theme
 
-> [!NOTE]
-> This is not your Windows Terminal profile. Your PowerShell profile is a script that runs every time PowerShell starts. [Learn more about PowerShell profiles](/powershell/module/microsoft.powershell.core/about/about_profiles).
+To browse the Oh My Posh prompt themes, use the command:
 
-### Install for PowerShell or WSL using Winget
+```powershell
+Get-PoshThemes
+```
 
-If you would like to use Oh My Posh to style both Windows and Windows Subsystem for Linux (WSL) command lines, we recommend using the [Windows Package Manager](/windows/package-manager). You can [install winget](https://github.com/microsoft/winget-cli#installing-the-client), the package manager client, using the App Installer from the Microsoft Store. (This may require an Insiders build if you're running Windows 10.)
+Choose a theme and update your PowerShell profile with this command. (You can replace `notepad` with the text editor of your choice.)
 
-1. Using PowerShell, install Oh My Posh using winget with the command:
+```powershell
+notepad $PROFILE
+```
 
-    ```powershell
-    winget install JanDeDobbeleer.OhMyPosh
-    ```
+Add the following to the end of your PowerShell profile file to set the `paradox` theme. (Replace `paradox` with the theme of your choice.)
 
-    This will install:
-    * oh-my-posh.exe - Windows executable, added to your $PATH.
-    * oh-my-posh-wsl - Linux executable, added to your $PATH for use in the WSL.
-    * themes - The latest Oh My Posh themes.
+```powershell
+Import-Module oh-my-posh
+Set-PoshPrompt -Theme paradox
```

Comment:
This cmdlet is currently deprecated. Favoring the init script instead:

```powershell
oh-my-posh --init --shell pwsh --config ~/jandedobbeleer.omp.json | Invoke-Expression
```

`Import-Module oh-my-posh` is only required when installing with `Install-Module oh-my-posh`

Reply:
I'm not sure that I understand and am not quite clear on what you're saying is deprecated. 
In the OMP docs, it looks like the PowerShell install for Windows has the first step for install is:  `Install-Module oh-my-posh -Scope CurrentUser`
...so I'm also confused by the statement that `Import-Module oh-my-posh` is only required when installing with `Install-Module oh-my-posh`.  We are installing with `Install-Module`, yes?

So the `Import-Module oh-my-posh` should _not_ be added to the PowerShell $PROFILE file?
And does that init script go in the $PROFILE to run every time? Or is this a one-time command?

Reply:
When installing with `Install-Module oh-my-posh` you always need to run `Import-Module oh-my-posh` to load the oh-my-posh executable in the `$PATH`. That part is correct. I was stating we've deprecated the `Set-PoshPrompt` cmdlet although it's still available and will work. Now, for sake of our documentation we're not adding that again as it makes the maintenance between everything rather challenging but in this context it makes perfect sense. So let's agree to keep supporting the cmdlet meaning the above information is correct.

Reply:
Ahh! Thanks for clarifying. Ok - so no change?

I can replace `Set-PoshPrompt` with `oh-my-posh --init --shell pwsh --config ~/jandedobbeleer.omp.json | Invoke-Expression` if that is the preferred method, but if it's just a matter of upkeep for documentation and `Set-PoshPrompt` works fine, then I'll leave as-is as you suggest. Just let me know if that changes and thank you again for the help on getting all of this clear.

Reply:
Let's leave it as is for the sake of simplicity.